core-bundle/src/Resources/contao/pages/PageRegular.php

```diff
@@ -213,7 +230,7 @@ protected function prepare($objPage)
 
 		// Assign the title and description
 		$this->Template->title = strip_tags($this->replaceInsertTags($objLayout->titleTag));
-		$this->Template->description = str_replace(array("\n", "\r", '"'), array(' ', '', ''), $objPage->description);
+		$this->Template->description = str_replace(array("\n", "\r", '"'), array(' ', '', ''), $responseContext->getMetaDescription());
```

Comment:
didn't we already sanitize the description when it was added to the response context?

Reply:
I think we should keep it as is because it's the way it was before. I guess if e.g. the news sets the description with its default process now it might contain newlines. I don't want to change that here now for BC reasons. But I agree that we should expect the context to be clean in future proper page controllers.